### DISCUSSION TITLE
Latest

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1774] [Fixed Issues]
Everytime I add an album , all my albums are shown in Latest

**Comment by klassikmann** [comment]
System: rAudio 202409141 + latest UPDATE ("Tag Editor" thread).

Using the "album view" all my albums are shown sorted by artist as specified in library options → o.k.
Using "Latest" all my albums are shown sorted by title → not o.k.

**Comment by rern** [comment]
Guys,

Please try the UPDATE.

**Reply by klassikmann** [reply]
> Please try the UPDATE.

Done. All my albums are still shown sorted by title but without thumbnails and clicking on an album has no effect.

**Reply by klassikmann** [reply]
Addendum: After adding new albums to the database "Latest" shows the correct new albums (not all albums as before).

**Comment by hjfisch** [comment]
After todays update , it seems to work
